test/unittests/stack_test.cpp

```diff
@@ -117,8 +117,6 @@ TEST(operand_stack, construct)
 {
     OperandStack stack(0);
     EXPECT_EQ(stack.size(), 0);
-    stack.shrink(0);
```

Comment:
Hm, this reminds me don't have a test for `stack.drop(0);`, it should be a no op. Perhaps just add here before `drop(1)`?